docs/usage/commands/setup.adoc

```diff
@@ -63,4 +64,19 @@ stackscli setup list
 ----
 
 .List out the configuration files
-image::images/stackscli-setup-list.png[]
\ No newline at end of file
+image::images/stackscli-setup-list.png[]
+
+===== Latest
+
+The way in which the CLI knows about the projects it can scaffold is by an embedded configuration file. As the Ensono Stacks ecosystem grows and new projects are added, the CLI will need to be updated to know about these new projects. The `latest` command will download the latest configuration file from the Ensono Stacks repository and place it in the correct location.
+
+This means that the CLI can be updated without needing to update the CLI itself.
+
+[source,powershell]
+----
+stackscli setup latest
+----
+
+This will result in the latest internal configuration file being downloads and placed in `$HOME/.stackscli/internal_config.yml`.
+
+NOTE: The CLI will always prefer this file over the embedded one. If the default behaviour is required then delete the downloaded file.
```

Comment:
You might wanna spit out a line in the CLI about which Config is being used so users have more of an idea if you're not already.

i.e.
```
stacks-cli foo
INFO: Using Internal Config file
```
Or
```
stacks-cli foo
INFO: Using External Config file at 'C:\blah\...'
```

Something along those lines

Reply:
@russellseymour Did you ever see his BTW?

Reply:
Yeah I did and I have made it more informative. I will add a screenshot to the PR.